xapian-applications/omega/templates/query

```diff
@@ -88,7 +88,7 @@ $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;
 <table>
 $if{$opt{clicklog},$set{queryid,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}}
 $hitlist{<tr><td valign=top>
-$if{$opt{clicklog},$set{clicklogurl,$html{$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}}}}}
+$if{$opt{clicklog},$set{clickurl,$or{$env{SCRIPT_NAME},omega.cgi}?FMT=clicklog&DB=$url{$dbname}&QUERYID=$opt{queryid}&DOCID=$url{$id}&URL=$url{$field{url}}},$set{clickurl,$html{$field{url}}}}
```

Comment:
Would this be clearer as `$set{clickurl,$if{$opt{clicklog},VIA-OMEGA,$field{url}}`?

In particular, it might make it clearer that the different legs here aren't equivalent: one still wraps in `$html{}`.